bundle/internal/schema/annotations_openapi_overrides.yml

```diff
@@ -230,6 +230,10 @@ github.com/databricks/cli/bundle/config/resources.Pipeline:
   "run_as":
     "description": |-
       PLACEHOLDER
+  "target":
+    "deprecation_message": This legacy field is deprecated for pipeline creation in favor of the schema field
+  "trigger":
+    "deprecation_message": Use continuous instead
```

Comment:
Qeustion -- is this file auto-generated or manually updated?

Reply:
This is manual because the OpenAPI schema doesn't have a message. We can automatically set "deprecated" for remaining schemas though without message: https://github.com/databricks/cli/pull/2652

Reply:
I do see information about deprecation in the docs though. Is the issue that it's not a separate field but part of the description?

https://docs.databricks.com/api/workspace/pipelines/create

> trigger
> Deprecated: Use continuous instead.

> target
> This legacy field is deprecated for pipeline creation in favor of the schema field.

Reply:
You can see it because text happens to be a part of description field, deprecation_message field has a special handing in VSCode and will show up on warning tooltip

Reply:
Docs website uses the same "deprecated" field as I was added in https://github.com/databricks/cli/pull/2652